run.sh

```diff
@@ -42,11 +45,16 @@ else
 fi
 
 if [ ! -e "$JTREG_HOME" ] ; then
-  ball=jtreg-7+1.tar.gz
+  ball=jtreg5.1-b01.tar.gz
   wget https://ci.adoptopenjdk.net/view/Dependencies/job/dependency_pipeline/lastSuccessfulBuild/artifact/jtreg/$ball
   tar -xf $ball
 fi
 
+AGENT_OPT=""
+if [ -n "${SKIP_AGENT_TESTS:-}" ] ; then
+    AGENT_OPT="-javaoption:-Dcryptotests.skipAgentTests=1"
+fi
+
```

Comment:
Maybe instead of this, some standard thing like JAVA_TOOL_OPTIONS or  similarly - generycally anything waht can be appended to?
Actually...JAVA_TOOL_OPTIONS jtreg dont like them, does ot?  So mabe  JAVA_OPTION apendable variable, with  some documentation of CryptoTests specific  -D proeprties?

Reply:
fair enough. thanx